images/scripts/fedora.setup

```diff
@@ -119,13 +119,15 @@ valkey \
 wireguard-tools \
 "
 
-ABRT_PACKAGES="\
-abrt \
-abrt-addon-ccpp \
-reportd \
-libreport-plugin-bugzilla \
-libreport-fedora
-"
+if [ "$VERSION_ID" -le "40" ]; then
+    ABRT_PACKAGES="\
```

Comment:
This is a bit awkward and also get into trouble due to `set -u`. I'd recommend to leave this definition as it is, and instead conditionalize the `dnf install` command. There's no problem with splitting up the `dnf install` command into two.

Reply:
ok, instead of splitting the `dnf install` command I added a condition where it adds ABRT packages to `ALL_DEPS`